system/mathLoadTest/playlist.xml

```diff
@@ -21,6 +21,7 @@
 		<testCaseName>MathLoadTest_all</testCaseName>
 		<variations>
 			<variation>NoOptions</variation>
+			<variation>-Xgcpolicy:gencon -Xgc:dynamicBreadthFirstScanOrdering</variation>
```

Comment:
Since this jvm option is specific openj9/ibm, please add the variation in the test target that specific to openj9/ibm. i.e., ClassLoadingTest_special. Please fix the rest of the PR